app/stac_api/admin.py

```diff
@@ -30,8 +31,21 @@ class CollectionLinkInline(admin.TabularInline):
 
 
 @admin.register(Collection)
-class CollectionAdmin(admin.ModelAdmin):
+class CollectionAdmin(admin.GeoModelAdmin):
+    Collection._meta.get_field('extent_geometry').editable = True  # to show the field
```

Comment:
Hmm, I think this will lead to problems if you change this on class level. I think you can either try to
- define the field in the admin with `editable=True` (e.g. sth like `extent_geometry = models.PolygonField(..., editable=True)
- or overwrite this info in the constructor of the class 
- or change it in the model (why is set to `editable=False` there if we need it editable=True?)

Reply:
That's sounds definitely a hack here. In the model this field should not be editable as well as in admin page. I agree with @boecklic having read only field in admin would be nice, but I would only do it if django allows it without any hack, so if there is any option in admin class to display non editable fields I would definitely leave it aside.

Reply:
yes it is a ugly hack to show the extent. But we need it editable=False in the model, as it is just an aggregation from items

Reply:
there is the option to display editable=False with readonly_fields = ['list_with_fields']. But for the PolygonField this results in a Testfield: Polygon(.....)... The hack is tho show the extent in the map.

Reply:
Have a look at https://docs.djangoproject.com/en/3.1/ref/contrib/admin/#django.contrib.admin.ModelAdmin.readonly_fields and https://stackoverflow.com/questions/48067882/django-admin-making-a-required-field-read-only. It seems there is a way to display read only fields without hack

Reply:
@ltshb I did this already in the PR line: 62